repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3585,6 +3478,25 @@ public List<String> propagateClassification(String entityGuid, String classifica
             AtlasVertex currentTaskVertex = (AtlasVertex) graph.query().has(TASK_GUID, currentTask.getGuid()).vertices().iterator().next();
             currentTaskVertex.setProperty(TASK_ASSET_COUNT_TO_PROPAGATE, impactedVertices.size());
 
+
+            //iterate over each impacted vertex to create a message and send it to kafka
+            for (AtlasVertex vertex : impactedVertices) {
+                // Create a Map to store vertex properties
+                Map<String, Object> vertexMap = new HashMap<>();
+                ObjectMapper objectMapper = new ObjectMapper(); // Jackson ObjectMapper
```

Comment:
Create objectMapper outside loop & reuse
Also, check if AtlasType.toJson() can be used instead of creating objectMapper